setup jdk24

olamy opened the pull request:
Signed-off-by: Olivier Lamy <olamy@apache.org>

olamy committed:
setup jdk24

Signed-off-by: Olivier Lamy <olamy@apache.org>

joakime commented:
Strange test failure on JDK24 ...

```
org.eclipse.jetty.server.CustomRequestLogTest.testLogRequestTimeCustomFormats([2] %{EEE MMM dd HH:mm:ss zzz yyyy|EST}t)

java.lang.AssertionError: 
Expected: a value less than or equal to <2L>
     but: <3600L> was greater than <2L>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
	at org.eclipse.jetty.server@12.0.18-SNAPSHOT/org.eclipse.jetty.server.CustomRequestLogTest.testLogRequestTimeCustomFormats(CustomRequestLogTest.java:625)
```

sbordet commented:
It is a daylight time issue... not sure why it fails only on Java 24... maybe the daylight configuration in Java 24 has been updated?

Ah yes:
https://bugs.openjdk.org/browse/JDK-8340477

sbordet committed:
Fixes test that was referring to deprecated timezone identifiers,
that have changed in Java 24 due to https://bugs.openjdk.org/browse/JDK-8340477.

Signed-off-by: Simone Bordet <simone.bordet@gmail.com>

olamy committed:
force groovy version to support jdk24

Signed-off-by: Olivier Lamy <olamy@apache.org>

olamy committed:
test if last invoker plugin version is enough

Signed-off-by: Olivier Lamy <olamy@apache.org>

olamy committed:
cleanup comments

Signed-off-by: Olivier Lamy <olamy@apache.org>
